src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name = "hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name = "connection.url">jdbc:mysql://localhost:3306/ticket_app?serverTimezone=UTC</property>
+        <property name = "connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name= "connection.username">root</property>
+        <property name= "connection.password">password1234</property>
+        <property name= "show_sql">true</property>
+        <property name= "hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.model.Movie" />
+        <mapping class="mate.academy.model.CinemaHall" />
+        <mapping class="mate.academy.model.MovieSession" />
+    </session-factory>
+</hibernate-configuration>
```

Comment:
done forget about empty line in the end of file 🙂

Reply:
okay)) thanks)))